proto/zitadel/settings/v2alpha/settings_service.proto

```diff
@@ -0,0 +1,343 @@
+        option (google.api.http) = {
+            get: "/v2alpha/settings/branding"
+        };
+
+        option (zitadel.protoc_gen_zitadel.v2.options) = {
+            auth_option: {
+                permission: "policy.read"
+            }
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Get the current active branding settings";
+            description: "Return the current active branding settings for the requested context"
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
+    rpc GetDomainSettings (GetDomainSettingsRequest) returns (GetDomainSettingsResponse) {
+        option (google.api.http) = {
+            get: "/v2alpha/settings/domain"
+        };
+
+        option (zitadel.protoc_gen_zitadel.v2.options) = {
+            auth_option: {
+                permission: "policy.read"
+            }
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Get the current active domain settings";
+            description: "Return the current active domain settings for the requested context"
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
+    rpc GetLegalSettings (GetLegalSettingsRequest) returns (GetLegalSettingsResponse) {
+        option (google.api.http) = {
+            get: "/v2alpha/settings/legal"
+        };
+
+        option (zitadel.protoc_gen_zitadel.v2.options) = {
+            auth_option: {
+                permission: "policy.read"
+            }
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Get the current active legal settings";
+            description: "Return the current active legal settings for the requested context"
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
+    rpc GetLockoutSettings (GetLockoutSettingsRequest) returns (GetLockoutSettingsResponse) {
+        option (google.api.http) = {
+            get: "/v2alpha/settings/lockout"
+        };
+
+        option (zitadel.protoc_gen_zitadel.v2.options) = {
+            auth_option: {
+                permission: "policy.read"
+            }
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Get the current active lockout settings";
+            description: "Return the current active lockout settings for the requested context"
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
+}
+
```

Comment:
In General all the messages below do not have openapi definitions

Reply:
do we really need additional definitions for those?
![image](https://user-images.githubusercontent.com/9405495/236477111-ec09f2e4-04b0-49d7-a0b9-105cc7068265.png)

![image](https://user-images.githubusercontent.com/9405495/236476809-2feb5ba1-7a62-4bfe-8fd8-787303102d4f.png)